Makefile

```diff
@@ -494,8 +494,13 @@ $$(UAVO_COLLECTION_DIR)/$(1)/uavo-xml: $$(UAVO_COLLECTION_DIR)/$(1)/uavo-xml.tar
 	$$(V1) rm -rf $$@
 	$$(V1) mkdir -p $$@
 	$$(V1) tar -C $$(call toprel, $$@) -xf $$(call toprel, $$<) || rm -rf $$@
+	$$(V1) python/dronin-canonicalize $$(call toprel, $$@)/shared/uavobjectdefinition
+	$$(V1) rm -f $$(call toprel, $$@)/shared/uavobjectdefinition/vtxconfig.xml
```

Comment:
Why rm this particular one?

Reply:
P.S. I see we renamed it... should remove all the underscores and ugliness in the field names while we're breaking it.

Reply:
UAVO has the same name-- but before UAVO names and filenames were unrelated.. canonicalization makes sure the filename is correct.

Reply:
(so wire format, actual stuff is all the same, but we need to remove the "original" file in this case because the canonicalization tool isn't smart enough to).

Reply:
> UAVO has the same name-- but before UAVO names and filenames were unrelated..

Ick. Good job. 👍